src/pyflex/config.py

```diff
@@ -32,8 +33,9 @@ def __init__(self, min_period, max_period, stalta_waterlevel=0.07,
                  snr_max_base=3.0, noise_start_index=0, noise_end_index=None,
                  signal_start_index=None, signal_end_index=-1,
                  window_weight_fct=None,
-                 window_signal_to_noise_type="amplitude",
-                 resolution_strategy="interval_scheduling"):
+                 window_signal_to_noise_type="energy",
+                 resolution_strategy="interval_scheduling",
+                 select_mode="body_and_surface"):
```

Comment:
Forget that. I read the code in the meanwhile and `body_and_surface` is the right choice.